app/Resources/views/procuration_manager/proposals.html.twig

```diff
@@ -27,29 +20,29 @@
 
     <hr />
 
-    {% if filters.hasData %}
-        {% if proxies|length == 0 %}
-            <div class="text--body text--center">
-                Aucun résultat
-            </div>
-        {% else %}
-            <div class="text--body text--center b__nudge--bottom-large">
-                {{ totalCount }} résultat(s)
-            </div>
-        {% endif %}
+    {% if 0 == proxies|length %}
+        <div class="text--body text--center">
+            {% if constant('AppBundle\\Procuration\\Filter\\ProcurationProxyProposalFilters::UNASSOCIATED') == filters.status %}
```

Comment:
If the variable `filters` is an instance of `ProcurationProxyProposalFilters` you can use it to do `constant('UNASSOCIATED', ProcurationProxyProposalFilters)`

And all constant in this file.